### DISCUSSION TITLE
[3 Novembre 2022] - Cache

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#49] [Cache]
### Traccia Esercizio

![243636818-01990f53-2d93-4287-9f1c-a4f994ab856f](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/25df7dd1-cfb2-472c-b7ae-939471984060)

**Comment by Divine-Sunderer** [comment]
<img width="2565" alt="3 Novembre 2022  - Cache" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/157913185/973599ce-1be8-4530-a615-a020527a8b0a">
